test/e2e/lib/install.bash

```diff
@@ -7,16 +7,16 @@ source "${E2E_DIR}/lib/template.bash"
 
 function install_tiller() {
   if ! helm2 version > /dev/null 2>&1; then # only if helm isn't already installed
-    kubectl --namespace kube-system create sa tiller
-    kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
-    helm2 init --service-account tiller --upgrade --wait
+    kubectl --namespace "$E2E_NAMESPACE" create sa tiller
+    kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount="$E2E_NAMESPACE":tiller
+    helm2 init --tiller-namespace "$E2E_NAMESPACE" --service-account tiller --upgrade --wait
   fi
 }
 
 function uninstall_tiller() {
-  helm2 reset --force
+  helm2 reset --tiller-namespace "$E2E_NAMESPACE" --force
```

Comment:
Can you add a note here about the `--force` flag, and the fact that this only resets the Tiller instance and _not_ the release storage.